conanfile.txt

```diff
@@ -0,0 +1,48 @@
+[package]
+name = up-client-vsomeip-cpp
+version = 0.1.1
+description = C++ vSomeip library implementing UTransport
+homepage = https://github.com/eclipse-uprotocol
+url = https://github.com/conan-io/conan-center-index
+license = Apache-2.0 license
+topics = ulink client, transport
+
+[settings]
+os
+compiler
+build_type
+arch
+
+[options]
+shared = [True, False]
+fPIC = [True, False]
+build_cross_compiling = [True, False]
+build_testing = [True, False]
+build_unbundled = [True, False]
+
+[default_options]
+shared = True
+fPIC = False
+build_cross_compiling = False
+build_testing = False
+build_unbundled = False
+
```

Comment:
`conanfile.txt` is not for building a package. I recommend checking the [conan documentation](https://docs.conan.io/2/reference/conanfile_txt.html).

```suggestion
```